### PR TITLE
Designed UI for Registration Page (Desktop)

### PR DESCRIPTION
**Pull request by rarchitgupta**
# Description

Coded React components and pages for registration page. Layout has to be changed completely for mobile, so haven't accounted for that yet

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by imabp** [comment]
Everything seems fine now @Souvikns in the PR

Fixes:
FontSize Hardcode Replaced with Chakra 'xl' version code
Padding Hardcode Replaced with Chakra 'xl' version code
Mobile Responsive Border Color Change
Mobile Responsive
Alignment Tests passed.

@Souvikns Do check the team page for this deploy preview [here](https://deploy-preview-110--techphantoms.netlify.app/team) , I think airtable servers are down or what ?

**Comment by imabp** [comment]
Hacktoberfest  has started. 
@rarchitgupta please check your hacktoberfest profile after this merge

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
-    "airtable": "^0.9.0",
+    "airtable": "^0.10.0",
```
Comment (imabp) [inline package.json:12]: @Souvikns this thing made that Issue raised : #113 . I have fixed it now.

Comment (imabp) [reply package.json:12]: @rarchitgupta please do not upgrade the package versions. It can seriously make applications crash.

Comment (rarchitgupta) [reply package.json:12]: Sorry about that. Didn't mean to

**Pull request merged**